pkg/platform/model/svc.go

```diff
@@ -38,7 +41,21 @@ func (m *SvcModel) EnableDebugLog() {
 
 func (m *SvcModel) request(ctx context.Context, request gqlclient.Request, resp interface{}) error {
 	defer profile.Measure("SvcModel:request", time.Now())
-	return m.client.RunWithContext(ctx, request, resp)
+
+	err := m.client.RunWithContext(ctx, request, resp)
+	if err != nil {
+		reqError := &gqlclient.RequestError{}
+		if errors.As(err, &reqError) {
+			logging.Debug(
+				"svc client gql request failed - query: %q, vars: %q",
+				reqError.Request.Query(),
+				jsonFromMap(reqError.Request.Vars()),
+			)
```

Comment:
This feels like it could easily end up leaking sensitive information via log files and rollbar. Maybe we can make it only report the vars for local builds and while running in tests?